lib/perl/Genome/Model/ClinSeq/Command/CreateMutationSpectrum.pm

```diff
@@ -59,6 +59,11 @@ class Genome::Model::ClinSeq::Command::CreateMutationSpectrum {
             default => 0,
         },
     ],
+    has_param => [
+        lsf_resource => {
+            default => "-R 'select[tmp>1000] span[hosts=1] rusage[tmp=1000]' -M 16000000",
```

Comment:
Does this step need 16GB of RAM?  If so, it should also indicate `mem=16000` in the select and rusage.  If not, the `-M` parameter can be removed.